bedrock/firefox/templates/firefox/features/safebrowser.html

```diff
@@ -0,0 +1,222 @@
+        store
+        logins or passwords on the cloud because it's more vulnerable to hacks.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Create secure passwords for every account and Password Manager will remember them all. We recommend creating a
+        Master
+        Password to safeguard all of your passwords in case your computer is lost or stolen.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Firefox Lockbox') }}</h2>
+      <p>
+        {% trans %}
+        You're carrying a fairly powerful tiny computer in your pocket,. You might know it as your phone. It's got its
+        own
+        internet browser, apps that connect to the web, passwords, and logins. You use it in coffee shops and other
+        public
+        places that make it easy for hackers to steal your information.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        On Firefox Lockbox, you can take your passwords with you and not worry about security. They're all one place and
+        protected on desktop, phone or tablet.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Tracking Protection') }}</h2>
+      <p>
+        {% trans %}
+        Being followed doesn't make anyone feel safe, even if it's an annoying ad following you on the web. Blocking
+        third-party
+        trackers can help by not allowing the annoying "cookies" tracking that enables ads.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        That's why we block third-party trackers and offer content blocking on private and regular browsing mode.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Private Browsing') }}</h2>
+      <p>
+        {% trans %}
+        Speaking of private browsing, it's a powerful privacy tool that's easy to use. On Firefox, private browsing
+        means we
+        don't save information about where you've been, clicked or looked. Private browsing leaves smaller digital
+        footprints,
+        giving away less about what you do and where you go online.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Containers') }}</h2>
+      <p>
+        {% trans %}
+        Cleaning up your on line space isn't just good feng shui, it can lessen the worry of being tracked. Separate
+        identities
+        for your work websites or your personal interests keep trackers confined to one area. Ads that have nothing to
+        do with
+        work won't pop-up in the middle of a presentation. For this purpose, we created Containers.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Containers help you stay organized and keep trackers in their place. Checking flight prices? Log onto the same
```

Comment:
Looks like you are missing links from the page copy. [The copy doc](https://docs.google.com/document/d/1C1n0x6YUXgkub07_EJiSB86u1PRQQU0mi_DzgRnT7zY/) shows that 'Containers' should link to https://addons.mozilla.org/firefox/addon/multi-account-containers/

We can do this in the following way:
```
{% trans url='https://addons.mozilla.org/firefox/addon/multi-account-containers/' %}
<a href="{{ url }}">Containers</a>...
```
`url` can be whatever variable name you would like. 🙂

Reply:
Keep in mind that we do not hard code locale in links (ex: en-US) so the user's locale is loaded.